idd/Energy+.idd.in

```diff
@@ -90281,10 +90283,10 @@ ElectricLoadCenter:Storage:Simple,
        \minimum 0.0
   N2,  \field Nominal Energetic Efficiency for Charging
        \maximum 1.0
-       \minimum 0.0
+       \minimum 0.001
```

Comment:
@RKStrand @mjwitte what do you think about this versus like `\minimum> 0`.  I'm not strongly opinionated either way.

Reply:
@Myoldmopar Well, the code that now exists uses 0.001 as a hard limit for both charging and discharging.  I guess we could eliminate that logic now with the change to the IDD.  If I remember correctly, my thinking was that it is going to divide by the number and I wanted to make sure we avoided the problem of having huge electric loads (an input file set this parameter to zero which ended up being a very tiny number and a massive electric load.  I guess I'm okay with it like this, but I'll wait to hear if @mjwitte has a strong opinion.

Reply:
No opinion, really. Anything below 0.1 seems pathetic anyway, so I would hope nobody actually models anything this low.

Reply:
:+1: